cookbooks/php/files/default/env.custom

```diff
@@ -0,0 +1,2 @@
+; Custom Environment variables should go here
```

Comment:
No need for that file, after https://github.com/engineyard/ey-cookbooks-stable-v6/pull/90/files is merged. Let's handle this on `env_vars` recipe for all cases/languages/webservers

Reply:
Pushed a commit that removed `cookbook_file "/data/#{app_name}/shared/config/env.custom"` and deleted files/default/env.custom.